bessctl/conf/testing/run_module_tests.bess

```diff
@@ -158,14 +162,11 @@ def monitor_task(module, wid):
 def run_tests():
     DID_TESTS_FAIL = False
 
-    path = os.path.dirname(os.path.realpath(sys.argv[0]))
-
     # Note: os.path.join will discard the leading parts if test
     # begins with '/', so callers should provide an absolute path
     # if and only if they want to override the internal BESS
     # path prefixing.
```

Comment:
This comment has become out of date.  It referred to this behavior:

    >>> os.path.join('/abs/path/to/dir', 'b')
    '/abs/path/to/dir/b'
    >>> os.path.join('/abs/path/to/dir', '/different/abs/b')
    '/different/abs/b'

 which reminds me of how I think I had changed it, I think I just let an abs path in `$TEST_PATH` override the whole thing, and you had to include the `.../module_tests` part.

Anyway, not really critical how it's done, but I think the first part is a bug.